Jenkinsfile

```diff
@@ -83,6 +83,13 @@ Boolean noOverwrites() {
     }
 }
 
+Boolean getBuildVariant() {
```

Comment:
Why not check for the env var `USE_SLE_STEMCELL`:
https://github.com/SUSE/scf/blob/develop/Jenkinsfile#L175